scripts/config.py

```diff
@@ -38,6 +38,7 @@ class Config:
     KEY_VERSION = 'CONFIG_VERSION'
     KEY_LINKERFORPROGRAMS = 'LINKERFORPROGRAMS'
     KEY_CXX = 'CXX'
+    KEY_LINKFLAGS = 'LINKFLAGS'
```

Comment:
This file was modified in the previous pull request. I do not see any overlapping code that changes the result, or breaks the compilation. But just to be sure, I prefer to comment it